.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,9 +14,9 @@ Our continues integration system doesn't send any notifications about failed tes
 
 Please provide a descriptive title of the pull request in the field 'Title' too.
 
-Please provide us information on how your pull request improves Pester or what fixes.
+Please provide us with information on how your pull request improves Pester or what fixes.
 
-If your pull request integrate Pester with another system (e.g. a continues integration product) please provide instruction how the updated code can be tested.
+If your pull request integrates Pester with another system (e.g., a continues integration product), please provide instruction how the updated code can be tested.
```

Comment:
The comma after e. g. is something I don't see used, so I don't think it belongs there. On the other hand CI should be spelled _Continuous_ integration, I think the same typo is in the contribution guide as well. Also instructions not instruction.